fiat-web/src/test/groovy/com/netflix/spinnaker/fiat/controllers/AuthorizeControllerSpec.groovy

```diff
@@ -31,6 +31,7 @@ import com.netflix.spinnaker.fiat.model.resources.Resource
 import com.netflix.spinnaker.fiat.permissions.PermissionsRepository
 import com.netflix.spinnaker.fiat.permissions.PermissionsResolver
 import com.netflix.spinnaker.fiat.providers.ResourcePermissionProvider
+import com.netflix.spinnaker.kork.web.exceptions.NotFoundException
```

Comment:
Unused?

Reply:
there previously was a c.n.s.f.controllers.NotFoundException (same package as this test) that I deleted in favor of the one from kork-exceptions